API Project/src/main/resources/config/application.properties

```diff
@@ -1,11 +0,0 @@
-# https://spring.io/guides/gs/accessing-data-mysql/
```

Comment:
While merging the branches, I accidentally deleted this file. Thank you for your comment.